iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisClient.java

```diff
@@ -167,4 +216,41 @@ public Action handleAttemptFailure(Event event) {
       return defaultPolicy.handleAttemptFailure(event);
     }
   }
+
+  // This policy is used to raft configuration change
+  //
+  private static class RatisEndlessRetryPolicy implements RetryPolicy {
+
+    private static final Logger logger = LoggerFactory.getLogger(RatisEndlessRetryPolicy.class);
+    private final RetryPolicy defaultPolicy;
+
+    RatisEndlessRetryPolicy() {
+      // about 1 hour wait Time.
+      defaultPolicy =
+          RetryPolicies.retryForeverWithSleep(TimeDuration.valueOf(2, TimeUnit.SECONDS));
+    }
+
+    @Override
+    public Action handleAttemptFailure(Event event) {
+      if (event.getCause() instanceof ReconfigurationInProgressException) {
```

Comment:
why not use if（A || B || C)

Reply:
haha!   I'm lazy here :) . This part is the copilot prompt, and the tab is generated with one click.

Reply:
Why not change this?

Reply:
I have changed